src/components/StickyResponsiveSidebar/StickyResponsiveSidebar.js

```diff
@@ -23,6 +36,9 @@ class StickyResponsiveSidebar extends Component {
     this._closeNavMenu = this._closeNavMenu.bind(this);
   }
 
+  _openNavMenu: Function;
+  _closeNavMenu: Function;
+
```

Comment:
Maybe we should just replace these lines:
```js
    this._openNavMenu = this._openNavMenu.bind(this);
    this._closeNavMenu = this._closeNavMenu.bind(this);
  }

  _openNavMenu: Function;
  _closeNavMenu: Function;

  _openNavMenu() {
    this.setState({open: !this.state.open});
  }

  _closeNavMenu() {
    this.setState({open: false});
  }
```

With this?

Reply:
Yes, absolutely. I didn't want to modify the actual code too much.

Reply:
Friendly ping.

Reply:
Hi, truly sorry! I've been a bit swamped, but I have a few more things I've done so I'll push those ASAP (probably Monday).